docformatter.py

```diff
@@ -38,6 +38,7 @@
 import sys
 import textwrap
 import tokenize
+from distutils.sysconfig import get_python_lib
```

Comment:
Could you try with standard `sysconfig` module instead of distutils?

Reply:
Sure, can do. Thx for the input